runtime/include/chpl-comm-diags.h

```diff
@@ -138,17 +138,19 @@ void chpl_comm_diags_verbose_printf(chpl_bool is_unstable,
   }
 }
 
-#define chpl_comm_diags_verbose_rdma(op, node, size, ln, fn)            \
+#define chpl_comm_diags_verbose_rdma(op, node, size, ln, fn, commid)     \
+  chpl_comm_diags_verbose_printf(false,                                  \
+                                 "%s:%d: remote %s, node %d, %zu bytes, " \
+                                 "commid %d",                            \
```

Comment:
Should be `"commid %" PRId32`, to match type of `commid`.  (and in the other instances)

Reply:
I cast it to `(int)` which is the same strategy used for the node printing.
I think casting and using `%d` is better because:
 * it still works correctly if the commid type changes
 * it's very unlikely that we will ever have more than 2**31 GET locations in a single file (well basically impossible really - the compiler would die producing such a file first). But anyway, if we did, the only bad effect would be that the comm id would print wrong (vs say accessing invalid memory)
 * it is shorter and less awkward than using `PRId32`.

Reply:
Sure, that's fine too.